crates/nu-cli/src/repl.rs

```diff
@@ -596,8 +602,9 @@ pub fn evaluate_repl(
                         PipelineData::empty(),
                         false,
                     );
+
+                    // Ensure bracketed paste is enabled again
                     if engine_state.get_config().bracketed_paste {
-                        #[cfg(not(target_os = "windows"))]
```

Comment:
This should probably stay because I don't believe bracketed paste is supported on Windows yet. IIRC, there was a crossterm issue about it, unless you're saying that's been fixed?

Reply:
> This should probably stay because I don't believe bracketed paste is supported on Windows yet. IIRC, there was a crossterm issue about it, unless you're saying that's been fixed?

I haven't looked into bracketed_paste much from the crossterm side. However it should be totally dependent on the terminal, not on the OS itself. And we can't assume that a windows user is using Windows Terminal.

This is why I asked for multiple people to test it on multiple platforms, to see if something breaks.

I don't like the solution of giving Windows users a configuration setting that is silently ignored either. Either they can or they can't configure bracketed paste.

Reply:
If bracketed paste isn't supported in a certain terminal (some unix terminals might not support it either) at its worst the characters are probably just ignored.

According to https://github.com/crossterm-rs/crossterm/issues/737, bracketed paste does work in WizTerm.

Reply:
I'm not super sure how to test this to ensure my buffer has the appropriate escape sequences when I paste them. But these seem to work on Windows Terminal and WezTerm.
```
❯ $"(ansi csi)200~this is a test(ansi csi)201~"
this is a test
❯ $"(ansi csi)?2004h(ansi csi)200~this is a test(ansi csi)201~(ansi csi)?2004l"
this is a test
```